psscriptanalyzer/Invoke-HealthCheck.ps1

```
@@ -0,0 +1,62 @@
+    Status    = "unknown"
+}
+
+function Check-Endpoints {
+  $endpoints = @(
+    "https://demoapp.example.com/health",
+    "https://api.demoapp.example.com/healthz",
+    "https://admin.demoapp.example.com/_status"
+  )
+  foreach ($url in $endpoints) {
+      $resp = iwr -Uri $url -UseBasicParsing
+      if ($resp.StatusCode -eq $null) {
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)